### DISCUSSION TITLE
Capture the screen of all clients every X seconds

### DISCUSSION THREAD

**Discussion by ricpelo** [epoptes/epoptes#98] [Q&A]
Hi! Sorry if this isn't the right place to ask this question, but I can't see where can I do it.

Is there a way to capture the screen of all clients every X seconds and save it in the teacher's computer?

Even better (although I suppose it's harder): is there a way to continuously capture the screen of all clients _in a video file_?

Thanks in advance! I love Epoptes, I use it in my classroom and it works very nicely :).

**Comment by alkisg** [comment]
Hi, Epoptes doesn't support saving screenshots or videos, you'd need an alternative solution like cron jobs to do that.

If someone wants to offer a patch, an "epoptes-exec" tool would be handy, that would run a command on the clients are fetch the results.
